profiles/models.py

```diff
@@ -22,4 +22,5 @@ def create_user_profile(sender, instance, created, **kwargs):
 
 post_save.connect(create_user_profile, sender=User)
 
-#User.profile = property(lambda u: u.profile())
+
+User.profile = property(lambda u: u.userprofile)
```

Comment:
@febinstephen This line was initially commented and not sure why it was commented now, can you please recheck it and let me know what it does ?